### PR TITLE
Write tests for Array class

### PR DESCRIPTION
**Pull request by yjaeseok**
I would like to write tests fro array class.

ISSUE=#332

### CODE REVIEW

File: test/basic_types/array.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+test(require(`../build/${buildType}/binding.node`));
+test(require(`../build/${buildType}/binding_noexcept.node`));
+
+function test(binding) {
+
+  // create empty array
+  const array = binding.basic_types_array.createArray();
+  assert.strictEqual(binding.basic_types_array.getLength(array), 0);
+
+  // create arrry with length
```
Comment (gabrielschulhof) [inline test/basic_types/array.js]: *array

File: test/basic_types/array.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  // check length after set data
+  assert.strictEqual(binding.basic_types_array.getLength(array), 3);
+
+  // get function test
+  assert.strictEqual(binding.basic_types_array.get(array, 0), 10);
+  assert.strictEqual(binding.basic_types_array.get(array, 1), "test");
+  assert.strictEqual(binding.basic_types_array.get(array, 2), 3.0);
+  
+  // overrite test
```
Comment (gabrielschulhof) [inline test/basic_types/array.js]: *overwrite

File: test/basic_types/array.js (hunk 1 above)
Comment (jjakeyoon) [reply test/basic_types/array.js]: thank you

**Comment by yjaeseok** [comment]
@mhdawson okay I will request PR after this patch.

**Comment by yjaeseok** [comment]
@mhdawson @gabrielschulhof please review it again. :)

**Comment by mhdawson** [comment]
Landed as 67b7db0a6fdf03f1c9e2e0c5d5a9bdd4851e92c1

**Pull request closed without merging**